pkgs/development/libraries/gettext/default.nix

```
@@ -52,7 +52,7 @@ stdenv.mkDerivation rec {
   strictDeps = true;
   nativeBuildInputs = [
     xz
-    xz.bin
+    (lib.getBin xz)
```

Isn't this automatically chosen here?

---

We don't need that here anymore. At the time the tarball was in the xz format but now we have gz and strictDeps is already set.

```suggestion
  buildInputs = [ bash libintl ]
```

The --with-xz seems to be not properly checked and for that to work we would need to move xz to buildInputs